### PR TITLE
update Quantinuum Target Availability descriptions

### PR DESCRIPTION
**Pull request by mlk621**
update Quantinuum Target Availability descriptions to make it clearer to users what they mean

### CODE REVIEW

**Comment by PRMerger-2** [comment]
@mlk621 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by tedhudek** [comment]
Docs Build status updates of commit _[3cb3ace](https://github.com/mlk621/quantum-docs/commits/3cb3ace33e7f85b939a4ca3b819faa8c1df5a272)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[articles/provider-quantinuum.md](https://github.com/mlk621/quantum-docs/blob/main/articles/provider-quantinuum.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://buildapi.docs.microsoft.com/Output/PullRequest/776a9f04-28cf-7639-37fb-77d3aae09b34/202208022115518842-560/BuildReport?accessString=642777c487c9234feca5ee240c60351b52dbe30a9adc37d8ac615076d357a802).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/776a9f04-28cf-7639-37fb-77d3aae09b34?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by mlk621** [comment]
@tedhudek - tagging you as you suggested! :)

**Comment by Jak-MS** [comment]
@Bradben 


Can you review this PR?

IMPORTANT: When this content is ready to merge, you must add `#sign-off` in a comment or the approval may get overlooked.


#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: articles/provider-quantinuum.md [hunk 1]
```diff
@@ -80,6 +70,24 @@ Both System Model H1 hardware H1-1 and H1-2 are continuously upgraded throughout
 
 Technical details for the System Model H1 and System Model H1 Emulators can be found in Quantinuum's [product data sheets](https://www.quantinuum.com/products/h1).
 
+## Target Availability
+
+The Quantinuum H-Series quantum computers were designed to be continuously upgraded, which allows customers to have access to the latest and greatest hardware capabilities as our teams continually improve gate fidelities, memory errors, and system speed.
```
Comment (Bradben) [inline articles/provider-quantinuum.md:75]:
```suggestion
The Quantinuum H-Series quantum computers are designed to be continuously upgraded, which allows customers to have access to the latest hardware capabilities as Quantinuum continually improves gate fidelities, memory errors, and system speed. 
```

File: articles/provider-quantinuum.md [hunk 2]
```diff
@@ -80,6 +70,24 @@ Both System Model H1 hardware H1-1 and H1-2 are continuously upgraded throughout
 
 Technical details for the System Model H1 and System Model H1 Emulators can be found in Quantinuum's [product data sheets](https://www.quantinuum.com/products/h1).
 
+## Target Availability
+
+The Quantinuum H-Series quantum computers were designed to be continuously upgraded, which allows customers to have access to the latest and greatest hardware capabilities as our teams continually improve gate fidelities, memory errors, and system speed. 
+
+The Quantinuum hardware cycles through commercial and development cycles. During commercial weeks, hardware is available to process jobs via a queue system. During development weeks, the hardware is offline as upgrades are completed.
```
Comment (Bradben) [inline articles/provider-quantinuum.md:77]:
```suggestion
Quantinuum hardware cycles through commercial periods and development periods. During commercial periods, the hardware is available to process jobs via a queue system. During development periods, the hardware is offline as upgrades are applied.
```

File: articles/provider-quantinuum.md [hunk 3]
```diff
@@ -80,6 +70,24 @@ Both System Model H1 hardware H1-1 and H1-2 are continuously upgraded throughout
...
+
+A calendar is sent to Quantinuum users monthly with information on the commercial and development cycles. If you have not received this please email QCsupport@quantinuum.com.
```
Comment (Bradben) [inline articles/provider-quantinuum.md:79]:
```suggestion
Every month, a calendar is sent to Quantinuum users with information on the commercial and development periods. If you have not received this calendar, please email QCsupport@quantinuum.com.
```

File: articles/provider-quantinuum.md [hunk 4]
```diff
@@ -80,6 +70,24 @@ Both System Model H1 hardware H1-1 and H1-2 are continuously upgraded throughout
...
+
+A target's status indicates its current ability to process jobs. The possible states of a target include:
+
+- **Available**: The target is online, processing submitted jobs and accepting new ones.
+- **Degraded**: The target is accepting jobs, but not currently processing them.
+- **Unavailable**: The target is offline, not accepting new job submissions.
+
+For the Quantinuum quantum computer targets, **Available** and **Degraded** correspond to commercial cycles, while **Unavailable** corresponds to development cycles while the machine is offline for upgrades.
```
Comment (Bradben) [inline articles/provider-quantinuum.md:87]:
```suggestion
For the Quantinuum quantum computer targets, **Available** and **Degraded** correspond to commercial periods, while **Unavailable** corresponds to development periods where the machine is offline for upgrades.
```

**Review by Bradben — APPROVED** [review]

**Comment by PRMerger7** [comment]
@Bradben : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by tedhudek** [comment]
Docs Build status updates of commit _[5a8ea7c](https://github.com/mlk621/quantum-docs/commits/5a8ea7c3626d615c60532622b82357089f1f98d0)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[articles/provider-quantinuum.md](https://github.com/mlk621/quantum-docs/blob/main/articles/provider-quantinuum.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://buildapi.docs.microsoft.com/Output/PullRequest/776a9f04-28cf-7639-37fb-77d3aae09b34/202208022310398271-560/BuildReport?accessString=e6e857b4a0433f7bcae31b3535a2c72013ea9752891f1ef5728dcb50d0e08e9e).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/776a9f04-28cf-7639-37fb-77d3aae09b34?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Review by Bradben — APPROVED** [review]

**Comment by Bradben** [comment]
#sign-off

**Comment by tedhudek** [comment]
Thanks again Megan - these updates should be on the live site within 5-10 minutes

**Comment by tedhudek** [comment]
Docs Build status updates of commit _[5a8ea7c](https://github.com/mlk621/quantum-docs/commits/5a8ea7c3626d615c60532622b82357089f1f98d0)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[articles/provider-quantinuum.md](https://github.com/mlk621/quantum-docs/blob/main/articles/provider-quantinuum.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://buildapi.docs.microsoft.com/Output/PullRequest/776a9f04-28cf-7639-37fb-77d3aae09b34/202208022347028624-560/BuildReport?accessString=70d533e3d91711a546cc5872c97d20392fb9ff1f5eb96518cd71a89087e68987).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/776a9f04-28cf-7639-37fb-77d3aae09b34?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Pull request merged**